js/pageView.js

```diff
@@ -276,7 +276,7 @@ define([
         removeClassFromHtml: function() {
```

Comment:
nothing to do with your update I know but not sure this really merits a function of its own since it's only one line and only referenced in one other place (immediately above)